Sources/SwiftCliCore/InputValidator.swift

```diff
@@ -26,4 +40,7 @@ public class InputValidator {
         }
         return(stitch_array)
     }
+    
+    public func CenteredArrayMaker(array: [[String]]) -> [[String]] {
+        return array}
```

Comment:
A little confused as to what this function is doing since it seems to just return the input without any transformation. Similarly, is anything calling this function?

Reply:
This is another leftover from last week, when I had a harebrained idea for creating all my increase-decrease features at one that involved positioning each row relative to an empty array that was based off the widest row. I'm going to delete this too, because after we talked on Friday, I realized a better way to do things. I was originally going to add some logic, but I see I was misguided.